lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/auth/OAuthRefreshTokenDaoImpl.java

```diff
@@ -252,4 +269,18 @@ public void deleteAllTokensForUserClientPair(String userId, String clientId) {
 		params.addValue(PARAM_CLIENT_ID, clientId);
 		namedParameterJdbcTemplate.update(DELETE_TOKEN_BY_CLIENT_USER_PAIR, params);
 	}
+
+	@Override
+	public void deleteLeastRecentlyUsedTokensIfOverLimit(String userId, String clientId, Long maxLeaseLengthInDays, Long maxNumberOfTokens) {
```

Comment:
As stated above, I think you can omit maxLeaseLengthInDays and just delete based on maxNumberOfTokens